test/modelValidation/swaggers/specification/parameterizedhost/face.json

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
should we simplify the spec even more? just to leave the cases you're checking for parameterized host and have simple models that don't require that many definitions, seems a bit unnecessary for the test case?

Reply:
I understand, but that is time consuming for now. I think we should leave it as-is.
I have added a test for semantic validation